.github/CODEOWNERS

```diff
@@ -4,8 +4,9 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-# The @googleapis/api-bigquery is the default owner for changes in this repo
-*               @googleapis/api-bigquery @googleapis/yoshi-python
 
-# The python-samples-reviewers team is the default owner for samples changes
-/samples/   @googleapis/python-samples-owners @googleapis/api-bigquery @googleapis/yoshi-python
+# @googleapis/yoshi-python and @googleapis/api-bigquery are the default owners for changes in this repo
+*     @googleapis/yoshi-python @googleapis/api-bigquery
+
+# @googleapis/python-samples-owners and @googleapis/api-bigquery are the default owners for samples changes
+/samples/   @googleapis/python-samples-owners @googleapis/api-bigquery
```

Comment:
It's been convenient to have `yoshi-python` here too, but understandable if `python-samples-owners` is response and/or the `api-bigquery` team can merge trivial changes like dependency updates.